fastlane_core/lib/fastlane_core/core_ext/shellwords.rb

```diff
@@ -0,0 +1,60 @@
+require_relative '../helper'
+require 'shellwords'
+
+# Here be monkey patches
+
+class String
+  # CrossplatformShellwords
+  def shellescape
```

Comment:
Does this 100% override the method defined by `shellwords` because this is required after `shellwords`?

Reply:
I am not sure what you are asking, but I mostly did this by how the other stuff was done - so possible I missed something.

My current understanding:
1. `fastlane_core/lib/fastlane_core.rb` requires `fastlane_core/core_ext/shellwords` which loads it from `/lib/fastlane_core/core_ext/shellwords.rb`.
2. There we require 'shellwords, which loads the native implementation.
3. Then the monkey patching happens.

Reply:
If your question is if this overwrites the `String.shellescape` defined by `Shellwords` completely, then: Yes. It completely deletes that method and replaces it with this code - which for non-Windows environments does the exact same thing as shellword's `String.shellescape` - handing the string over to `Shellwords.escape`.

Reply:
Ah yes, your assumption of what I was asking in your last response was correct 👌 Thanks for the answer! ❤️